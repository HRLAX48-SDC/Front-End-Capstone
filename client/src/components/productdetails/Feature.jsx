import React, {useState, useEffect} from 'react';

function Feature(props) {
  return (
    <div>
      {props.features.map(feature => (
        <div key={feature.feature}>{feature.feature}: {feature.value}</div>
      ))}
    </div>
  );
}

export default Feature;