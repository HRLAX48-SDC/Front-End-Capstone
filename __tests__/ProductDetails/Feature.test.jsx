import React from 'react';
import renderer from 'react-test-renderer';
import Feature from '../../client/src/components/productdetails/Feature.jsx';

test('renders Feature component correctly', () => {
  const feature = renderer.create(<Feature
    features={[{feature: 'Fabric', value: 'Canvas'}]}
  />).toJSON();
  expect(feature).toMatchSnapshot();
});