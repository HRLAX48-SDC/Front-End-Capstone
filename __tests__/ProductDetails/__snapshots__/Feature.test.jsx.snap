// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Feature component correctly 1`] = `
<div>
  <div>
    Fabric
    : 
    Canvas
  </div>
</div>
`;
